.github/workflows/azure_cd_example.yml

```
@@ -0,0 +1,59 @@
+          version: 0.221.1
+
+      - name: AZ CLI login
+        uses: azure/login@a65d910e8af852a8061c627c456678983e180302
+        with:
+          tenant-id: ${{ env.AZURE_TENANT_ID }}
+          subscription-id: ${{ env.AZURE_SUBSCRIPTION_ID }}
+          client-id: ${{ env.AZURE_CLIENT_ID }}
+          enable-AzPSSession: false #enable powershell false
+
+      - name: Set Databricks Token
+        run: echo "DATABRICKS_TOKEN=$(az account get-access-token --resource 2ff814a6-3304-4ab8-85cb-cd0e6f879c1d | jq .accessToken --raw-output)" >> $GITHUB_ENV
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:138: Double quote to prevent globbing and word splitting

(shellcheck)